theme.config.jsx

```diff
@@ -3,8 +3,9 @@ import React from 'react';
 export default {
   logo: <span>Igbo API Documentation</span>,
   project: {
-    link: 'https://github.com/shuding/nextra',
-    icon: <span>🇳🇬</span>,
+    link: 'https://github.com/nkowaokwu/igbo_api',
+    icon: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" width="24" height="24"><path d="M12 .297c-6.627 0-12 5.373-12 12 0 5.302 3.438 9.8 8.205 11.385.6.111.82-.258.82-.577 0-.285-.011-1.04-.017-2.042-3.338.725-4.042-1.611-4.042-1.611-.546-1.385-1.334-1.754-1.334-1.754-1.089-.742.083-.727.083-.727 1.204.085 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.304 3.495.998.109-.776.417-1.305.76-1.605-2.665-.305-5.466-1.334-5.466-5.93 0-1.31.465-2.38 1.235-3.219-.125-.304-.535-1.524.116-3.176 0 0 1.005-.322 3.3 1.23.96-.266 1.98-.399 3-.405 1.02.006 2.04.139 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.652 1.652.242 2.872.117 3.176.765.839 1.23 1.91 1.23 3.219 0 4.608-2.805 5.623-5.475 5.918.43.371.81 1.105.81 2.227 0 1.608-.015 2.905-.015 3.3 0 .32.21.695.825.577 4.764-1.585 8.198-6.083 8.198-11.385 0-6.627-5.373-12-12-12z" /></svg>,
+    // icon: <span>🇳🇬</span>,
```

Comment:
let's revert this change and keep the nigeria flag - we can create a new PR that makes this icon change